apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestAudioActivity.java

```diff
@@ -906,8 +1092,12 @@ public synchronized void closeAudio() {
             }
         }
 
+        if (isAudioFocusEnabled()) {
+            AudioManager am = (AudioManager) getSystemService(Context.AUDIO_SERVICE);
```

Comment:
A more practical way is to cache the audio manager instead of getting it every time when closing.